src/index.html

```diff
@@ -11,7 +11,175 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+    >
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="field">
+          <span>Surname:</span>
+          <input
+            type="text"
+            name="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label  class="field">
+          <span>Name:</span>
+          <input
+          type="text"
+          name="name"
+          required
+          autocomplete="off"
+          >
```

Comment:
previous input has proper attributes indentation, 
this one does not
make sure to scan your code and apply everywhere: attributes (if multi-lined) should be indented +1 level vs the containing tag
```suggestion
          <input
            type="text"
            name="name"
            required
            autocomplete="off"
          >
```